crates/shared/src/signature_validator.rs

```diff
@@ -10,6 +10,10 @@ use hex_literal::hex;
 use primitive_types::H160;
 use thiserror::Error;
 
+lazy_static::lazy_static! {
+    static ref TRANSACTION_INITIALIZATION_GAS_AMOUNT: U256 = U256::from_dec_str("21000").unwrap();
+}
```

Comment:
We've been using `u64` for gas units generally because it is easier to work with (I see it is not consistent above too, but let's still do it this way for the new code). I'd make this that type of constant too so we don't need lazy static. So in the function where the constant is used either convert the existing U256 to u64 or convert the constant.